.vscode-config/launch.json

```diff
@@ -52,4 +52,4 @@
             "preLaunchTask": "build",
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
think we decided to remove this file from the repo in the past. Surprised to see it!

Reply:
These are not the default `.vscode` files.

They were added by @danewalton to help customers copy/paste them, if needed, here: https://github.com/Azure/azure-sdk-for-c/pull/968

See https://github.com/Azure/azure-sdk-for-c#vscode

Reply:
yea these are fine. they are stored in the `.vscode-config` directory and are meant to be copied over to the `.vscode` directory by the user.